nixos/modules/hardware/video/nvidia.nix

```diff
@@ -576,6 +576,8 @@ in
               "nvidia"
               "nvidia_modeset"
               "nvidia_drm"
+              # See https://github.com/NixOS/nixpkgs/issues/334180
+              (lib.optionalString cfg.open "nvidia_uvm")
             ];
```

Comment:
This adds an empty string to the list when `!cfg.open`; you must've meant `++ optionals cfg.open [ "nvidia_uvm" ]`

Reply:
You're right, I'll change it in a bit.